src/main/java/com/redhat/red/build/finder/json/KojiBuild.java

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.redhat.red.build.finder;
+package com.redhat.red.build.finder.json;
```

Comment:
I guess I don't understand why this file should move. This file is critical to the code in general and not specific to JSON.

Reply:
I _thought_ it was streamed by JSON but we can put all the code in one place ; I have no strong opinions on that. I can revert the subdirectory if you prefer

Reply:
Well, it is, but then so is nearly every class. So, in that sense the separation isn't really going to make any sense.